knowledge_repo/app/templates/index-base.html

```diff
@@ -98,12 +82,51 @@
 <script type="text/javascript">
 
     $( document ).ready(function() {
-     $.ajaxSetup({
-      cache: true
-    });
+      $.ajaxSetup({
+        cache: true
+      });
+
+      function getParameters() {
```

Comment:
I responded to this above - the `feed_params` don't always have all the information required.